backend/src/repository/promotion.repository.ts

```diff
@@ -0,0 +1,32 @@
+import { getRepository } from 'typeorm';
+import { PROMOTION_DB_ERROR } from '../constants/database-error-name';
+import { Promotion } from '../entity/Promotion';
+import { DatabaseError } from '../errors/base.error';
+
+async function createPromotion(imgUrl: string): Promise<Promotion> {
+  try {
+    const promotionRepo = getRepository(Promotion);
+    const newPromotionEntity = promotionRepo.create({
+      imgUrl,
+    });
+    return await promotionRepo.save(newPromotionEntity);
```

Comment:
혹시 아래처럼 하면 동작을 안하는걸까요??
```suggestion
    return await promotionRepo.save({imgUrl});
```

Reply:
아니요 말씀하신거랑 똑같아요 말씀하신게 더 나을 수 있습니다.

create는 다른 object를 딥카피해서 새로 생성하는 걸로알고 잇어요.